packages/block-library/src/audio/edit.js

```diff
@@ -116,7 +116,7 @@ function AudioEdit( {
 	}
 	if ( ! src ) {
 		return (
-			<Block.div>
+			<div { ...blockProps() }>
```

Comment:
Misplaced `()` here:
```suggestion
			<div { ...blockProps }>
```